packages/contracts-bedrock/contracts/L2/L2StandardBridge.sol

```diff
@@ -151,10 +151,22 @@ contract L2StandardBridge is StandardBridge, Semver {
     ) external payable virtual {
         if (_l1Token == address(0) && _l2Token == Predeploys.LEGACY_ERC20_ETH) {
             finalizeBridgeETH(_from, _to, _amount, _extraData);
+            emit DepositFinalized(_l1Token, _l2Token, _from, _to, _amount, _extraData);
         } else {
-            finalizeBridgeERC20(_l2Token, _l1Token, _from, _to, _amount, _extraData);
+            bool finalized = finalizeBridgeERC20(
+                _l2Token,
+                _l1Token,
+                _from,
+                _to,
+                _amount,
+                _extraData
+            );
+            if (finalized) {
+                emit DepositFinalized(_l1Token, _l2Token, _from, _to, _amount, _extraData);
```

Comment:
The abstract contract emits a failed event so I'm hesitant with accepting this change. Curious what the thinking is here

Reply:
This particular event is emitted by the existing L2 bridge. It's for backwards compatibility.